boards/shields/esp_8266/doc/index.rst

```diff
@@ -76,16 +81,57 @@ to works:
         - ESP-8266 bootloader won't send garbage. Try connect at 74880 bps if
           you module have 26MHz crystal to detect boot fails.
 
-Programming
-***********
+.. note::
+	The shield configuration was designed to be used with multiple network
+	interfaces.  This means, boards with any other network interface can
+	be simultaneous enabled.  User must be check network documentation to
+	avoid problem when running samples.  For instance, boards with ethernet
+	automatically define that interface as default.  Make sure to select
+	WiFi interface as the default interface when build examples.
+	As general rule, keep only WiFi interface enabled at
+	Networking -> Link Layer Options.  This will ensure samples works as
+	expected.
+
+Supported variations
+====================
+
+The below table suggests shield variation accordingly with end user
+application.  The default shield esp_8266 expects a board overlay.  The
+remaining configurations should be used based on the board standard headers
+available.  Any variation selects automatically the especific overlay that
+define right configuration.
```

Comment:
I think this last sentence should be removed, since what you describe is valid for all the shield.
```suggestion
available.
```

Reply:
done.